src/doc/cel.cpp

```diff
@@ -54,9 +54,7 @@ Cel* Cel::MakeCopy(const frame_t newFrame,
 Cel* Cel::MakeLink(const frame_t newFrame,
                    const Cel* other)
 {
-  Cel* cel = new Cel(newFrame, other->dataRef());
-  cel->setZIndex(other->zIndex());
-  return cel;
+  return new Cel(newFrame, other->dataRef());
```

Comment:
Revert this, we can copy the z-index of the given cel and adjust later if needed.